packages/jest-each/.npmignore

```diff
@@ -0,0 +1,3 @@
+**/__tests__/**
```

Comment:
We can add `**/__mocks__/**` just in case

Reply:
yeah, this should copy all of the other npmignores, but add `asset` in addition to them